jinahub/encoders/image/ImageTorchEncoder/README.md

```diff
@@ -2,24 +2,57 @@
 
 **ImageTorchEncoder** wraps the models from [torchvision](https://pytorch.org/vision/stable/index.html).
 
-**ImageTorchEncoder** encodes `Document` blobs of type a `ndarray` and shape Batch x Height x Width x Channel 
-into a `ndarray` of Batch x Dim and stores them in the `embedding` attribute of the `Document`.
+**ImageTorchEncoder** encodes `Document` blobs of type `ndarray` and shape Height x Width x Channel 
+into an `ndarray` of shape `embedding_dim` and stores them in the `embedding` attribute of the `Document`.
 
 
+## Usage
 
 
-## Usage
+Use the prebuilt images from Jina Hub in your Python codes and add it to your Flow:
+```python
+from jina import Flow
+
+flow = Flow().add(uses='jinahub+docker://ImageTorchEncoder')
+```
+### Encoding example
+After creating a Flow, prepare your Documents to encode. They should have the blob attribute set with shape 
+Height x Width x Channel. Then, we can start the Flow and encode the Documents. By default, any endpoint will encode 
+the Documents:
+
+```python
+import numpy as np
+
+from jina import Flow, Document
+
+f = Flow().add(uses='jinahub+docker://ImageTorchEncoder')
+
+doc = Document(blob=np.ones((224, 224, 3), dtype=np.uint8))
 
+with f:
+    resp = f.post(on='/index', inputs=doc, return_results=True)
+    print(f'{resp}')
+    
+    
+print('\n\nembedding:\n\n', resp[0].docs[0].embedding)
```

Comment:
Can you create some lambda to print the embedding - `return_results` is discouraged

Reply:
done